Documentation/troubleshooting.rst

```diff
@@ -326,6 +326,19 @@ Command to collect the system dump using the cluster-diagnosis tool:
 
     python cluster-diagnosis.zip sysdump
 
+You can specify from which nodes to collect the system dumps by passing
```

Comment:
I think this is good to have for now, but I think we should have documentation for the tooling in its own repo (e.g., in `cilium/cluster-diagnosis`) and link to it from here. Otherwise we will fall out of sync eventually.

Reply:
Agreed re getting out of sync, but the `--nodes` param is very often used by users in trouble. Also, passing host instead of IP addr seems to result in partial dumps, so it's good to remind to use IP addrs here.